cmake_format/test/cmake-format.yaml

```diff
@@ -13,6 +13,10 @@ separate_ctrl_name_with_space : false
 # If true, separate function names from the parenthesis with a space
 separate_fn_name_with_space : false
 
+# list of directories which will be ignore during parsing
```

Comment:
@dkandyba `ignore -> ignored`

Reply:
@AByzhynar Fixed in: dc880b1cb71405b990ba4615189e39ccbd14f6a6